### PR TITLE
Commit after cloning the repo

### PR DESCRIPTION
**Pull request by hassanShakur**
Initial commit after coning the portfolio repo.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+
+/* ============ End Contact Section ============== */
+
+/* ============ Footer Section ============== */
+
+footer {
+  margin: 2rem 0;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
```
Comment (hassanShakur) [inline style.css:388]: Great job! Just remember it's always nice to use relative values that are responsive as opposed to absolute ones.

File: index.html
Comment (hassanShakur) [inline index.html:1]: Good job!!

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @hassanShakur, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job using Github inline comments feature :+1: 
- You phrase your comments politely :100: 

## Required Changes ♻️
- [ ] Nice work using GitHub inline comments feature:clap: However it would be nice to add a comment stating the status of the project ( if you approve it or not). Watch this [short video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) for more information.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hassanShakur — COMMENTED** [review]
# Approved 👌 
Your project is complete! There is nothing else to say other than... it's time to merge.
Congratulations 👏 .

**Review by skyv26 — CHANGES_REQUESTED** [review]
Hi @hassanShakur, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] Please adhere to the project requirements, according to the project requirement you had to make at least two inline comments but you made only one `inline-comment`. Kindly take a look at the below-attached image.

Screenshot from 2023-05-01 16-49-19

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+  min-height: 92vh;
+  padding: 5%;
+  padding-right: 8%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1rem;
+  position: relative;
+  color: #fff;
+}
```
Comment (hassanShakur) [inline style.css:109]: Nice work on the background image positioning. Love it.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+
+.skills {
+  color: #fff;
+  font-size: 22px;
+  font-family: 'Inter', sans-serif;
+  display: flex;
+  flex-direction: column;
+  gap: 1.4rem;
+  align-items: center;
+  padding: 10rem 0;
```
Comment (hassanShakur) [inline style.css:315]: Awesome work on this. Just as a suggestion, remember to use relative dimensions when working with CSS to ensure your website remains responsive.

**Review by hassanShakur — COMMENTED** [review]
# PR Approved 🥂 
Your project is complete! There is nothing else to say other than... it's time to merge.
Just keep in mind what is included in the comments and make the corrections if you please.

**Review by aamirkhan2478 — APPROVED** [review]
Hi @hassanShakur , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Good use of inline comments :+1: 
✅ Well done you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**